test/Interpreter/SDK/archiving_generic_swift_class.swift

```diff
@@ -190,10 +190,10 @@ func unarchive() {
     fatalError("unable to unarchive Foo<NSNumber>")
   }
 
-  // CHECK-LABEL: Foo<NSString>
+  // CHECK-LABEL: <_TtGC4main3FooCSo8NSString_: {{0x[0-9a-f]+}}> #0
```

Comment:
Agreed. I'll have to look into that specific case, but in general getting rid of `summary` has made some of the descriptions less legible. I posted a thread about it in swift-evolution last week, but it's currently inactive.